pyproject.toml

```
@@ -9,3 +9,20 @@ __version__ = "{version}"
 """
 fallback_version = "0.0"
 
+[tool.black]
+exclude = '''
+# Directories
+/(
+    \.eggs
+  | \.git
+  | \.hg
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | venv
+  | _build
+  | build
+  | dist
+)/
+'''
+line-length = 79
```

I would recommend 99 characters.

```suggestion
line-length = 99
```

---

My bad, I thought I changed it to match the flake8 config (99), somehow missed it. Thanks.